deploy/charts/cert-manager/templates/rbac.yaml

```
@@ -401,10 +444,33 @@ subjects:
 
 ---
 
+{{ if .Values.global.rbac.disableHTTPChallengesRole }}
```

suggestion: Shouldn't this be not?

```suggestion
{{ if not .Values.global.rbac.disableHTTPChallengesRole }}
```